integrations/openai/src/index.ts

```diff
@@ -309,7 +309,7 @@ export default new bp.Integration({
       try {
         response = await openAIClient.audio.speech.create(params)
       } catch (err: any) {
-        throw new UpstreamProviderError(err.message, err)
+        throw new RuntimeError(err.message, err, undefined, {})
```

Comment:
**question:** Shouldn't this be the upstream error exported from common?

Reply:
Good catch, [addressed here](https://github.com/botpress/botpress/pull/13537/commits/5c5f4b4043edd814ab6b1b86b29a20c7e116f9e0#diff-619e23493a3bf6bd3d5fc3069317a6964c7f1143287715debe6e5d4c1e031fbaR312)